From: Hao Ge <gehao@kylinos.cn>
Subject: mm/pagemap: make trylock_page return bool
Date: Sun, 28 Apr 2024 09:47:11 +0800

Make trylock_page return bool to align the return values of folio_trylock
function and it also corresponds to its comment.

Link: https://lkml.kernel.org/r/20240428014711.11169-1-gehao@kylinos.cn
Signed-off-by: Hao Ge <gehao@kylinos.cn>
Cc: Matthew Wilcox (Oracle) <willy@infradead.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/pagemap.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/pagemap.h~mm-pagemap-make-trylock_page-return-bool
+++ a/include/linux/pagemap.h
@@ -1011,7 +1011,7 @@ static inline bool folio_trylock(struct
 /*
  * Return true if the page was successfully locked
  */
-static inline int trylock_page(struct page *page)
+static inline bool trylock_page(struct page *page)
 {
 	return folio_trylock(page_folio(page));
 }
_
